rhel/SPECS/dbaas-kubectls.spec

```diff
@@ -0,0 +1,100 @@
+%global with_debug   0
+
+%if 0%{?with_debug}
+# https://bugzilla.redhat.com/show_bug.cgi?id=995136#c12
+%global _dwz_low_mem_die_limit 0
+%else
+%global debug_package %{nil}
+%endif
+
+%global commit                  v1.16.8
```

Comment:
That line will be sed'ed by `prepare_specs` in `build-server-rpm`. That's not how we want it to work. It also should not work at all. It if does (even after bumping release version version to bypass S3 cache), that's _very_ concerning

Reply:
Updated **build-server-rpm** script